src/reducers/notesReducerFunction.js

```diff
@@ -82,6 +86,25 @@ const notesReducerFunction = (
 				labels: [...prevNotesState.labels, { label, id: labelId }],
 			};
 
+		case actionTypes.FILTER_BY_LABELS:
+			return {
+				...prevNotesState,
+				filterByLabel,
+			};
+
+		case actionTypes.SORT_BY:
+			return {
+				...prevNotesState,
+				sortBy,
+			};
+
+		case actionTypes.RESET_FILTERS:
+			return {
+				...prevNotesState,
+				sortBy,
+				filterByLabel,
+			};
+
 		default:
 			return prevNotesState;
 	}
```

Comment:
throw an error saying "action type not found", there might be a case where you may pass wrong action type and the app would still work because it's returning the prev state.